src/utils/beacon/payloadDecoder.ts

```diff
@@ -0,0 +1,45 @@
+import { hex2buf } from "@taquito/utils";
+
+const PAYLOAD_PADDING = "0501";
```

Comment:
please add a link where you took it from (either taquito or beacon would suffice)